applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -98,6 +98,9 @@ For example:
 
 For information about how to upgrade the device firmware using a PC or a smartphone, see the :ref:`matter_bridge_app_dfu` section.
 
+.. note::
+    Currently the DFU over Bluetooth LE SMP and Matter OTA are not available for the nrf54h20 board target.
```

Comment:
This is not a board target, a board target is the full device name e.g. nrf54h20dk/nrf54h20/cpuapp, not nrf54h20, that is just a board